index.html

```diff
@@ -0,0 +1,308 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <link rel='icon' href='./Bilder/favicon.png' type='image/x-icon'/>
+    <title>noemi's vinyls</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Peddana&family=Permanent+Marker&display=swap" rel="stylesheet"> 
+    <style> html, body{ 
+      scroll-behavior: smooth;
+      }</style>
+    <!-- dont forget the meta tag for responsiveness -->
+    </head>
+
+  <body>
+    <div class="title-area">
+      <div class="introduction"><h1>noemi's vinyl collection</h1>
+        <p class="introduction">finally i got around to inventory my vinyls! 
+        the main-purpose here is to keep myself from buying the same vinyl twice. seems simple, but happened already twice.</p>
+        </div>
+      </div>
```

Comment:
Indentation - the closing tag should be on the same level of indentation as the opening tag:
```
<div class="title-area">
  <div class="introduction"><h1>noemi's vinyl collection</h1>
    <p class="introduction">finally i got around to inventory my vinyls! the main-purpose here is to keep myself from buying the same vinyl twice. seems simple, but happened already twice.</p>
  </div>
</div>
```

Reply:
Plus, here you have 2 spaces indentation